sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1437,6 +1437,15 @@ object SQLConf {
     .intConf
     .createWithDefault(20)
 
+  val FAST_HASH_AGGREGATE_MAX_ROWS_CAPACITY_BIT =
+    buildConf("spark.sql.fast.hash.aggregate.row.max.capacity.bit")
```

Comment:
This name looks too long. How about `spark.sql.codegen.aggregate.map.row.capacitybit` or others?